### PR TITLE
i18n: dashboard settings

### LINKED ISSUE #75531: i18n: Mark up `DashboardSettings` for translations
follow instructions [here](https://github.com/grafana/grafana/blob/main/contribute/internationalization.md#how-to-add-a-new-translation-phrase). can verify translation strings have been marked up correctly by running `yarn i18n:pseudo` and using the development-only pseudo locale: 
image

section title checklists:
- [ ] Settings
- [ ] General
- [ ] Annotations
- [ ] Variables
- [ ] Links
- [ ] Versions
- [ ] Permissions
- [ ] JSON Model

### PR DESCRIPTION
**Pull request by abhiShandy**
**What is this feature?**

[Add a brief description of what the feature or update does.]

**Why do we need this feature?**

[Add a description of the problem the feature is trying to solve.]

**Who is this feature for?**

[Add information on what kind of user the feature is for.]

**Which issue(s) does this PR fix?**:

Fixes #75531

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: public/app/features/dashboard/components/DashboardSettings/DashboardSettings.tsx [hunk 1]
```diff
@@ -135,7 +137,7 @@ function getSettingsPages(dashboard: DashboardModel) {
 
   if (dashboard.id && dashboard.meta.canSave) {
     pages.push({
-      title: 'Versions',
+      title: t('dashboard-settings.page-titles.versions', 'Versions'),
```
Comment (ashharrison90) [inline public/app/features/dashboard/components/DashboardSettings/DashboardSettings.tsx:140]:
```suggestion
      title: t('dashboard-settings.versions.title', 'Versions'),
```

File: public/app/features/dashboard/components/DashboardSettings/DashboardSettings.tsx [hunk 2]
```diff
@@ -185,7 +187,7 @@ function getSectionNav(
   location: H.Location
 ): NavModel {
   const main: NavModelItem = {
-    text: 'Settings',
+    text: t('dashboard-settings.page-titles.settings', 'Settings'),
```
Comment (ashharrison90) [inline public/app/features/dashboard/components/DashboardSettings/DashboardSettings.tsx:190]:
```suggestion
    text: t('dashboard-settings.settings.title', 'Settings'),
```

File: public/app/features/dashboard/components/DashboardSettings/DashboardSettings.tsx [hunk 3]
```diff
@@ -161,7 +163,7 @@ function getSettingsPages(dashboard: DashboardModel) {
   }
 
   pages.push({
-    title: 'JSON Model',
+    title: t('dashboard-settings.page-titles.json-model', 'JSON Model'),
```
Comment (ashharrison90) [inline public/app/features/dashboard/components/DashboardSettings/DashboardSettings.tsx:166]:
```suggestion
    title: t('dashboard-settings.json-editor.title', 'JSON Model'),
```

**Review by ashharrison90 — COMMENTED** [review]
generally looks great!

couple of comments about adjusting the naming style and also extracting out any translations used more than once into variables.

any questions, gimme a shout 👍

**Comment by abhiShandy** [comment]
> generally looks great!
> 
> couple of comments about adjusting the naming style and also extracting out any translations used more than once into variables.
> 
> any questions, gimme a shout 👍

I have made the requested changes.

File: public/app/features/dashboard/components/DashboardSettings/DashboardSettings.tsx [hunk 4]
```diff
@@ -161,7 +166,7 @@ function getSettingsPages(dashboard: DashboardModel) {
   }
 
   pages.push({
-    title: 'JSON Model',
+    title: t('dashboard-settings.json.title-model', 'JSON Model'),
```
Comment (ashharrison90) [inline public/app/features/dashboard/components/DashboardSettings/DashboardSettings.tsx:169]:
```suggestion
    title: t('dashboard-settings.json-editor.title', 'JSON Model'),
```

**Review by ashharrison90 — COMMENTED** [review]
we want this one to match the existing key that's already there. otherwise just needs main merging in then looks good 👍

**Review by ashharrison90 — APPROVED** [review]
@abhiShandy when you merged you inadvertently deleted all the translations! not sure how you managed that, but i rebased your branch and fixed it for you 😅 

looks great now 🙌 thanks for the contribution 🥳

**Comment by abhiShandy** [comment]
I ran `yarn i18n:extract` to resolve conflicts, and I didn't check the differences :(

**Pull request merged**